### ISSUE TITLE
Version 5.7 is broken

### ISSUE THREAD

**Issue by xzenon** [Pushwoosh/pushwoosh-ios-sdk#70]
With version 5.7 installed application can not be compiled anymore.

To test I just created new empty project and use the following Podfile:

```
source 'https://github.com/CocoaPods/Specs.git'

platform :ios, '10.0'

use_frameworks!

target ‘TestPushwoosh’ do
    pod 'Pushwoosh’, ‘5.7’
end
```
I got the following error when trying to build the project:
![pushwoosh-issue](https://user-images.githubusercontent.com/2015453/39924459-8a97457e-5530-11e8-9528-6b35c6214080.jpg)

When switched to version 5.6 - it builds without issues:

```
source 'https://github.com/CocoaPods/Specs.git'

platform :ios, '10.0'

use_frameworks!

target ‘TestPushwoosh’ do
    pod 'Pushwoosh’, ‘5.6’
end
```

As I can see, in version 5.7 modulemap file is renamed from **Pushwoosh.framework/Modules/module.modulemap** to **Pushwoosh.framework/Modules/pw.modulemap**. Probably it is the reason why framework could not be properly initegrated into the project.

**Comment by wfhm** [comment]
Hi,

Checking it.

**Comment by wfhm** [comment]
@xzenon Fixed in [5.7.1](https://github.com/Pushwoosh/pushwoosh-ios-sdk/tree/5.7.1)
